### PR TITLE
fix flakey test, add video.plcmt

### PR DESCRIPTION
**Pull request by nickjacob**

### CODE REVIEW

File: test/open_rtb_ecto/v2/bid_request_test.exs [hunk 1]
```diff
@@ -163,7 +163,11 @@ defmodule OpenRtbEcto.V2.BidRequestTest do
 
   describe "json encoding" do
     test "fields with nil values are omitted" do
-      assert "{\"at\":2,\"test\":0,\"allimps\":0}" == Jason.encode!(%BidRequest{})
+      str = Jason.encode!(%BidRequest{})
```
Comment (sneako) [inline test/open_rtb_ecto/v2/bid_request_test.exs:166]: ah yeah, the default map sorting changed in otp 26 :)

File: lib/open_rtb_ecto/v2/bid_request/video.ex [hunk 2]
```diff
@@ -37,6 +37,7 @@ defmodule OpenRtbEcto.V2.BidRequest.Video do
     field(:mincpmpersec, :float)
     field(:slotinpod, :integer)
     field(:placement, :integer)
+    field(:plcmt, :integer)
```
Comment (sneako) [inline lib/open_rtb_ecto/v2/bid_request/video.ex:40]: what version adds this field? I dont see it in 2.5 or 2.6

Comment (sneako) [reply lib/open_rtb_ecto/v2/bid_request/video.ex:40]: ah i see here: https://iabtechlab.com/blog/march-2023-update-to-openrtb-is-now-ready-for-implementation/

Comment (nickjacob) [reply lib/open_rtb_ecto/v2/bid_request/video.ex:40]: https://iabtechlab.com/blog/updates-to-openrtb-for-public-comment/
https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/develop/AdCOM%20v1.0%20FINAL.md#list_plcmtsubtypesvideo

Comment (nickjacob) [reply lib/open_rtb_ecto/v2/bid_request/video.ex:40]: https://iabtechlab.com/blog/march-2023-update-to-openrtb-is-now-ready-for-implementation/

**Pull request merged**